DOCSP-44135: Mention vector search in pymongo search indexes page (#116)
mongodb/docs-pymongo: source/indexes/atlas-search-index.txt

You can use the `create_search_index() <{+api-root+}pymongo/collection.html#pymongo.collection.Collection.create_search_index>`__
and the
`create_search_indexes() <{+api-root+}pymongo/collection.html#pymongo.collection.Collection.create_search_indexes>`__
methods to create Atlas Search indexes or Atlas Vector Search indexes.

The following code example shows how to create a single Atlas Search index:

.. literalinclude:: /includes/indexes/indexes.py
   :language: python
   :start-after: start-create-search-index
   :end-before: end-create-search-index

> The following code example shows how to create a single Atlas Vector Search index

Reviewer: ```suggestion
The following code example shows how to create a single Atlas Vector Search index by
```

using the `SearchIndexModel <{+api-root+}pymongo/operations.html#pymongo.operations.SearchIndexModel>`__
object:

.. literalinclude:: /includes/indexes/indexes.py
   :language: python
   :start-after: start-create-vector-search-index
   :end-before: end-create-vector-search-index

> You can use `create_search_indexes() <{+api-root+}pymongo/collection.html#pymongo.collection.Collection.create_search_indexes>`__

Reviewer: ```suggestion
You can use the `create_search_indexes() <{+api-root+}pymongo/collection.html#pymongo.collection.Collection.create_search_indexes>`__ method
```

to create multiple indexes. These indexes can be Atlas Search indexes or Atlas

> Vector Search indexes. ``create_search_indexes()`` requires the ``SearchIndexModel``

Reviewer: ```suggestion
Vector Search indexes. The ``create_search_indexes()`` method takes a list of ``SearchIndexModel``
```

> object.

Reviewer: ```suggestion
objects that correspond to each index you want to create.
```

You can use the
`list_search_indexes() <{+api-root+}pymongo/collection.html#pymongo.collection.Collection.list_search_indexes>`__

> method to return the Atlas Search indexes and Atlas Vector Search indexes 
> of a collection.

Reviewer: ```suggestion
method to return information about the the Atlas Search indexes and Atlas Vector Search indexes 
of a collection.
```

You can use the
`update_search_index() <{+api-root+}pymongo/collection.html#pymongo.collection.Collection.update_search_index>`__

> method to update an Atlas Search index or Atlas Vector Search index.

Reviewer: ```suggestion
method to update an Atlas Search or Vector Search index.
```

The following code example shows how to update an Atlas Search index:

.. literalinclude:: /includes/indexes/indexes.py
   :language: python
   :dedent:
   :start-after: start-update-search-indexes
   :end-before: end-update-search-indexes

The following code example shows how to update an Atlas Vector Search index:

.. literalinclude:: /includes/indexes/indexes.py
   :language: python
   :dedent:
   :start-after: start-update-vector-search-indexes
   :end-before: end-update-vector-search-indexes

.. _pymongo-atlas-search-index-drop:

Delete a Search Index
---------------------

You can use the
`drop_search_index() <{+api-root+}pymongo/collection.html#pymongo.collection.Collection.drop_search_index>`__

> method to remove an Atlas Search index or Atlas Vector Search index.

Reviewer: ```suggestion
method to remove an Atlas Search or Vector Search index.
```
